packages/wonder-blocks-dropdown/components/dropdown.js

```diff
@@ -12,15 +12,41 @@ import Spacing from "@khanacademy/wonder-blocks-spacing";
 import {View} from "@khanacademy/wonder-blocks-core";
 
 import visibilityModifierDefaultConfig from "../util/visibility-modifier.js";
-import typeof ActionItem from "./action-item.js";
-import typeof OptionItem from "./option-item.js";
-import typeof SeparatorItem from "./separator-item.js";
+import SeparatorItem from "./separator-item.js";
+import type {DropdownItem} from "../util/types.js";
+
+const keyCodes = {
+    tab: 9,
+    escape: 27,
+    space: 32,
+    up: 38,
+    down: 40,
+};
 
 type DropdownProps = {|
     /**
      * Items for the menu.
      */
-    children: Array<React.Element<ActionItem | OptionItem | SeparatorItem>>,
+    items: Array<DropdownItem>,
+
+    /**
+     * An index that represents the index of the focused element when the menu
+     * is opened.
+     */
+    initialFocusedIndex: number,
```

Comment:
Is this to handle the case where you've already selected something from the dropdown and you're opening the dropdown a second time?

Reply:
It's to handle the case for single select menus, where the focused item on open should be the selected one, if any.